CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 ## Unreleased
--
+- Fix BitBoxApp crash on GrapheneOS by declaring Google Play Services in the AndroidManifest.xmls, and adding ML Kit barcode scanning as a dependency in build.gradle.
```

Comment:
this changelog is user facing, so it should be light on tech details. Just "Fix BitBoxApp crash on GrapheneOS and other phones without Google Play Services when scanning QR codes" or similar will do.

Reply:
thanks.